cni/network/network.go

```diff
@@ -651,6 +667,22 @@ func (plugin *NetPlugin) createNetworkInternal(
 		ServiceCidrs:                  ipamAddConfig.nwCfg.ServiceCidrs,
 	}
 
+	ipv4Subnet := network.SubnetInfo{
+		Family:  platform.AfINET,
+		Prefix:  *podSubnetPrefix,
+		Gateway: ipamAddResult.ipv4Result.IPs[0].Gateway,
+	}
+	nwInfo.Subnets = append(nwInfo.Subnets, ipv4Subnet)
+
+	if ipamAddResult.ipv6Result != nil && len(ipamAddResult.ipv6Result.IPs) > 0 {
+		ipv6Subnet := network.SubnetInfo{
+			Family:  platform.AfINET6,
+			Prefix:  *podSubnetV6Prefix,
+			Gateway: ipamAddResult.ipv6Result.IPs[0].Gateway,
+		}
+		nwInfo.Subnets = append(nwInfo.Subnets, ipv6Subnet)
+	}
+
 	setNetworkOptions(ipamAddResult.ncResponse, &nwInfo)
 
 	addNatIPV6SubnetInfo(ipamAddConfig.nwCfg, ipamAddResult.ipv6Result, &nwInfo)
```

Comment:
is this still required? i see anyway you are adding ipv6 subnet in nwinfo above?

Reply:
need to discuss this

Reply:
This is going to add ipv6 subnet when it's ipv6nat mode, so it should be there since we did not add ipv6routes as ipv6nat mode

Reply:
it does the same thing. lets remove that.anyway we are not supporting ipv6nat mode